scrapy/utils/log.py

```diff
@@ -145,6 +145,9 @@ def write(self, buf):
         for line in buf.rstrip().splitlines():
             self.logger.log(self.log_level, line.rstrip())
 
+    def flush(self):
+        pass
```

Comment:
Should it flush logger handlers? 

http://stackoverflow.com/a/18012843/114795 suggests it can be done using
` [h.flush() for h in my_logger.handlerList]`.